packaging/hip-targets.cmake

```diff
@@ -47,6 +50,9 @@ endif()
 get_filename_component(_DIR "${CMAKE_CURRENT_LIST_DIR}" REALPATH)
 get_filename_component(_IMPORT_PREFIX "${_DIR}/../../../" REALPATH)
 
+# The installation prefix configured by this project.
+set(_IMPORT_PREFIX "${CMAKE_INSTALL_PREFIX}")
```

Comment:
This is called by other packages so `CMAKE_INSTALL_PREFIX` will not be set to hip's installation path. A relative path should be used.

Reply:
For example, the onnx runtime will call `find_package(hip)` but the `CMAKE_INSTALL_PREFIX` will be set to `/usr/local` or some other directory. It will never be set to hip's installation path.

Reply:
Removed the hard overwrite of `_IMPORT_PREFIX`

Reply:
Thanks!